### PR TITLE
Added additional permission for KSQL RBAC access for interactive queries

### PR DESCRIPTION
**Pull request by JumaX**
# Description

This PR corrects the following:

grant ResourceOwner on _confluent-ksql-<ksql-id>  topic (currently none)

grant ResourceOwner on _confluent-ksql-<ksql-id> subject (currently DeveloperRead)

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Successfully validated against rbac-scram-custom-rhel scenario.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: roles/confluent.ksql/tasks/rbac.yml [hunk 1]
```diff
@@ -85,6 +85,11 @@
             "resourceType":"Group",
             "name":"_confluent-ksql-{{ksql_final_properties['ksql.service.id']}}",
             "patternType":"PREFIXED"
+          },
+          {
+            "resourceType":"Topic",
+            "name": "_confluent-ksql-{{ksql_final_properties['ksql.service.id']}}",
```
Comment (stan-confluent) [inline roles/confluent.ksql/tasks/rbac.yml:91]: With this in place, there's no need to grant permissions to the `_confluent-ksql-{{ksql_final_properties['ksql.service.id']}}_command_topic` topic above (line 76) - but doesn't have to be done in this PR

File: roles/confluent.ksql/tasks/rbac.yml [hunk 2]
```diff
@@ -85,6 +85,11 @@
...
+            "patternType": "LITERAL"
```
Comment (stan-confluent) [inline roles/confluent.ksql/tasks/rbac.yml:92]: This should be PREFIXED - apologies, this one is on me, my jira was not clear enough

File: roles/confluent.ksql/tasks/rbac.yml (hunk 1 above)
Comment (JumaX) [reply roles/confluent.ksql/tasks/rbac.yml:86]: Removed.

File: roles/confluent.ksql/tasks/rbac.yml (hunk 2 above)
Comment (JumaX) [reply roles/confluent.ksql/tasks/rbac.yml]: Updated.

**Review by stan-confluent — APPROVED** [review]

**Review by stan-confluent — APPROVED** [review]

**Pull request merged**